pkgs/purs/src/CirclesPink/Garden/Env.purs

```diff
@@ -55,6 +56,19 @@ env { request, envVars } =
         web3 <- mapExceptT liftEffect $ getWeb3 envVars
         circlesCore <- mapExceptT liftEffect $ getCirclesCore web3 envVars
         account <- mapExceptT liftEffect $ CC.privKeyToAccount web3 privKey
+        safeAddress <- getSafeAddress privKey
+        --------------------------------------------------------------------------------
+        -- This Section cannot go to production! it auto-funds safe of user on register
+        --------------------------------------------------------------------------------
+        _ <-
```

Comment:
This will most likely be buggy in our production app. We should test it with the real address for safe funding though, to see what happens.

Reply:
This could be the "real" one: 0x450Fc3eAc8b847a759Cb05790cE7A1f465ac0cE8 as it funded my personal Safe

Address:
https://blockscout.com/xdai/mainnet/address/0x450Fc3eAc8b847a759Cb05790cE7A1f465ac0cE8/transactions

Transaction funding my safe:
https://blockscout.com/xdai/mainnet/tx/0x9e710c2767d5e1cc7b769b20741c3792d0c3b2391eb0ed5c0108534936485c41